14.serial_search/serial_search.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'date'
+
+def serial_search(input)
+  year = 1900 + (input[0] + input[4]).to_i
+  days = input[1..3].to_i
+  production = input[5..7].to_i
+
+  first_date = Date.new(year)
+  manufactured = first_date + (days - 1)
```

Comment:
```suggestion
  manufactured_on = first_date + days - 1
```

日付なら `_on`、日時なら `_at`にすることが多いですね（主にRails界隈の慣習ですが）

あと丸かっこはなくてもいいのでは？